### PR TITLE
wavefront_python_sdk renamed to wavefront_sdk

### PR DESCRIPTION
**Pull request by ruivapps**
wavefront_python_sdk renamed to wavefront_sdk
update setup.py and wavefront_pyformance/wavefront_reporter.py

### CODE REVIEW

**Comment by ustinov** [comment]
@vikramraman I already have PR #10 addressing half of it, need to consolidate these two

**Comment by ruivapps** [comment]
No problem.

**Comment by ustinov** [comment]
@ruivapps, the PR #10 just merged, please rebase this one on top of it. Thanks!

**Comment by ustinov** [comment]
LGTM
NB: if you rebased your PR on top of current master, instead of merging, you would end up only having one line diff

**Comment by ruivapps** [comment]
I did rebase and push. from my side. it looks fine. 

git diff upstream/master

```
diff --git a/setup.py b/setup.py
index b086e38..e1ed491 100644
--- a/setup.py
+++ b/setup.py
@@ -15,6 +15,7 @@ except ImportError:
     import sys
     sys.exit('Failed to import setuptools. Do pip install setuptools first.')

+REQUIRES = ["pyformance >= 0.4", "wavefront-sdk-python"]

 setuptools.setup(
     name='wavefront_pyformance',
diff --git a/wavefront_pyformance/wavefront_reporter.py b/wavefront_pyformance/wavefront_reporter.py
index 7bc8656..7d60491 100644
--- a/wavefront_pyformance/wavefront_reporter.py
+++ b/wavefront_pyformance/wavefront_reporter.py
@@ -4,7 +4,7 @@
 from __future__ import unicode_literals
 import json
 from pyformance.reporters import reporter
-from wavefront_python_sdk import WavefrontDirectClient, WavefrontProxyClient
+from wavefront_sdk import WavefrontDirectClient, WavefrontProxyClient
 from . import delta

 try:
```

**Comment by ustinov** [comment]
I see, looks like you missed the version requirement for wavefront-sdk-python

File: setup.py [hunk 1]
```diff
@@ -1,37 +1,36 @@
...
-# prerequisite: setuptools
-# http://pypi.python.org/pypi/setuptools
+try:
+    import setuptools
+except ImportError:
+    import sys
+    sys.exit('Failed to import setuptools. Do pip install setuptools first.')
 
-REQUIRES = ["pyformance >= 0.4", "wavefront_python_sdk"]
+REQUIRES = ["pyformance >= 0.4", "wavefront-sdk-python"]
```
Comment (ustinov) [inline setup.py]: Remove this line

**Review by ustinov — APPROVED** [review]

**Comment by ruivapps** [comment]
I did following 

```
git checkout master
git pull upstream
git merge upstream/master
git checkout  ISSUE/wavefront_python_sdk_rename
git rebase master 
resolve conflict 
git add setup.py
git dif upstream/master
git push
```

maybe I didn't do it right. I am noob on github. it's my first time rebase from different upstream source on github.

**Comment by ruivapps** [comment]
I see the issue. it's during the conflict resolution. I didn't pay enough attention that the requires moved to setuptools.setup.

from the issue branch 
```
git diff upstream/master
diff --git a/wavefront_pyformance/wavefront_reporter.py b/wavefront_pyformance/wavefront_reporter.py
index 7bc8656..7d60491 100644
--- a/wavefront_pyformance/wavefront_reporter.py
+++ b/wavefront_pyformance/wavefront_reporter.py
@@ -4,7 +4,7 @@
 from __future__ import unicode_literals
 import json
 from pyformance.reporters import reporter
-from wavefront_python_sdk import WavefrontDirectClient, WavefrontProxyClient
+from wavefront_sdk import WavefrontDirectClient, WavefrontProxyClient
 from . import delta

 try:
```

**Comment by ruivapps** [comment]
I don't have permission to merge it. 
feel free to merge it when you have time

**Pull request merged**